mava/systems/tf/maddpg/system.py

```diff
@@ -58,10 +65,10 @@ def __init__(
         ] = training.MADDPGDecentralisedTrainer,
         executor_fn: Type[core.Executor] = MADDPGFeedForwardExecutor,
         num_executors: int = 1,
-        num_caches: int = 0,
```

Comment:
Do we not support trainer caching anymore? Or how does the caching work in this version?

Reply:
The caching never worked or did it? I don't mind adding it back in, but just removed it because caching just crashed if you set it to 1.